Add more template fields to `DbtBaseOperator` (#786)
astronomer/astronomer-cosmos: docs/configuration/operator-args.rst

            "skip_exit_code": 1,
        }
    )

Template fields
---------------

Some of the operator args are `template fields <https://airflow.apache.org/docs/apache-airflow/stable/howto/custom-operator.html#templating>`_ for your convenience.

These template fields can be useful for hooking into Airflow `Params <https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html>`_, or for more advanced customization with `XComs <https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/xcoms.html>`_.

The following operator args support templating, and are accessible both through the  ``DbtDag`` and ``DbtTaskGroup`` constructors in addition to being accessible standalone:

- ``env``
- ``vars``

> - ``full_refresh`` (for the ``build``, ``seed``, and ``run`` operators.)

Reviewer: ```suggestion
- ``full_refresh`` (for the ``build``, ``seed``, and ``run`` operators since Cosmos 1.4.)
```

.. note::
    Using Jinja templating for ``env`` and ``vars`` may cause problems when using ``LoadMode.DBT_LS`` to render your DAG.

> The following template fields are only selectable when using the operators in a standalone context:

Reviewer: ```suggestion
The following template fields are only selectable when using the operators in a standalone context (starting in Cosmos 1.4):
```

- ``select``
- ``exclude``
- ``selector``
- ``models``

> The aforementioned args are not available to be templated via ``DbtDag`` and ``DbtTaskGroup`` because they need to select dbt nodes to render the DAG's tasks.

Reviewer: ```suggestion
Since Airflow resolves template fields during Airflow DAG execution and not DAG parsing,  the args above cannot be templated via ``DbtDag`` and ``DbtTaskGroup`` because both need to select dbt nodes during DAG parsing. 
```

> Since template fields are rendered on each ``DagRun``,

Reviewer: ```suggestion
```
